### DISCUSSION TITLE
隐藏文章一直loading中无法加载

### DISCUSSION THREAD

**Discussion by cao-gift** [tangly1024/NotionNext#2065] [General]
**描述bug**
隐藏文章无法通过设置slug正常访问

**复现步骤**
链接 https://120707.xyz/cs 一直loading中

但是可以通过Notion那里看到的文章ID 通过 https://120707.xyz/4491b49cfb194f5a8b26f65d7280e4e0 正常访问

**期望的正常结果**
正常加载

**截图**
问题图
![问题图](https://upload-bbs.miyoushe.com/upload/2023/04/15/283684029/c8a1d167cec0754a70d682c30a923557_6590611954164600820.png)

Notion相关设置
![Notion相关设置](https://upload-bbs.miyoushe.com/upload/2023/04/15/283684029/29d8ee7de40d8bd512f6c9eab291ec77_5969981241665295216.png)

**Comment by tangly1024** [comment]
文章正确的访问路径：
https://120707.xyz/article/cs
